src/pages/home/report/ReportActionCompose/ComposerWithSuggestionsEdit/ComposerWithSuggestionsEdit.tsx

```diff
@@ -0,0 +1,102 @@
+import type {Dispatch, ForwardedRef, RefObject, SetStateAction} from 'react';
+import React, {useState} from 'react';
+import type {MeasureInWindowOnSuccessCallback, TextInput} from 'react-native';
+import Composer from '@components/Composer';
+import type {ComposerProps} from '@components/Composer/types';
+import type {SuggestionsRef} from '@pages/home/report/ReportActionCompose/ReportActionCompose';
+import Suggestions from '@pages/home/report/ReportActionCompose/Suggestions';
+
+type Selection = {
+    start: number;
+    end: number;
+};
+
+type ComposerWithSuggestionsEditProps = ComposerProps & {
+    setValue: Dispatch<SetStateAction<string>>;
+    setSelection: Dispatch<SetStateAction<Selection>>;
+    resetKeyboardInput: () => void;
+    isComposerFocused: boolean;
+    suggestionsRef: RefObject<SuggestionsRef>;
+    updateDraft: (newValue: string) => void;
+    measureParentContainer: (callback: MeasureInWindowOnSuccessCallback) => void;
+    value: string;
+    selection: Selection;
```

Comment:
`ComposerProps` type already has `value`, `selection` and `isGroupPolicyReport` properties